Library/Homebrew/tap.rb

```diff
@@ -722,7 +722,11 @@ def formula_renames
   # Hash with tap migrations.
   sig { returns(Hash) }
   def tap_migrations
-    @tap_migrations ||= if (migration_file = path/HOMEBREW_TAP_MIGRATIONS_FILE).file?
+    @tap_migrations ||= if name == "homebrew/cask" && !Homebrew::EnvConfig.no_install_from_api?
+      migrations, = Homebrew::API.fetch_json_api_file "homebrew_cask_tap_migrations.json",
```

Comment:
Need to double-check after https://github.com/Homebrew/brew/pull/15628 is merged that this:
- returns data in the format that's expected
- caches as expected/very aggressively

Reply:
> * returns data in the format that's expected

I did, it did.

> * caches as expected/very aggressively

Bumped a stale check time instead.